sys/fido2/ctap/ctap_mem.c

```diff
@@ -154,3 +166,13 @@ int fido2_ctap_mem_get_offset_of_rk_into_flashpage(uint16_t rk_idx)
 
     return -1;
 }
+
+unsigned fido2_ctap_mem_get_state_flashpage_number(void)
+{
+    return flashpage_page((void *)_backing_memory);
+}
+
+unsigned fido2_ctap_mem_get_rk_start_page(void)
```

Comment:
You want this function explicitly to have an abstraction `+1` (rk flashpage offset)? Does it need to be exposed or would static inline do?